Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md

```diff
@@ -134,6 +134,17 @@ Following policy will configure/Audit security contacts (email) at Subscription
 
 ___ 
 
+#### Policy Assessment Evaluation
+
+To collect the complete audit evaluation for a control, please run the policies in below order.
+
+The assessment result should be combination of:
+
+1. [Security - SQL Server - AuditSqlServerAuditingSettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerAuditingSettings.json)
+2. [Security - SQL Server - AuditSqlServerSecuritySettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerSecuritySettings.json)
+3. [Security - SQL Server - AuditSqlServerThreatDetection](Security%20-%20SQL%20Server%20-%20AuditSqlServerThreatDetection.json) OR [Security - Subscription - UpdateSecurityContacts](Security%20-%20Subscription%20-%20UpdateSecurityContacts.json)
+___ 
+
 
 ### Notes
 1. It is recommended to assign policy to setup MDC security contacts at Subscription scope (or Management group with subscriptions managed by same team) as it will configure same email address for all subscriptions in scope.
```

Comment:
1. It is recommended to assign the policy to audit the MDC security contacts configuration at Subscription scope (or Management group with subscriptions managed by the same team) as the policy will configure the same email address for all subscriptions in a given MG scope.

I hope that is what you meant by highlighting the "scope" dimension?

Reply:
Yes Zia, highlighted the Audit thing